plugins/action_generate_review_assignment_records/src/generateReviewAssignments.test.ts

```diff
@@ -4,7 +4,7 @@
 // database has entered correct information
 
 import { map, omit } from 'lodash'
-import DBConnect from '../../../src/components/databaseConnect'
+import DBConnect from '../../../src/components/database/databaseConnect'
```

Comment:
Should have been part of previous PR (#1122), but didn't notice until I tried to build and got errors